src/types.ts

```diff
@@ -52,7 +52,6 @@ export interface KaboomCtx {
 	 *     area(),
 	 *     body(),
 	 *     health(8),
-	 *     doubleJump(),
```

Comment:
Could be a user-defined comp /shrug

Reply:
but it's confusing, since body() already has a doubleJump function